web/webkit/src/main/scala/net/liftweb/http/Templates.scala

```diff
@@ -28,7 +28,10 @@ import java.io.InputStream
  * Contains functions for obtaining templates
  */
 object Templates {
-  private val suffixes = LiftRules.templateSuffixes
+  // Making this lazy to ensure it doesn't accidentally init before Boot completes in case someone touches this class.
+  private lazy val parsers = LiftRules.contentParsers.flatMap(parser =>
```

Comment:
Good questions and suggestions.  My gut feel is we should make the last one win, so it's easier to override the default parsers without declaring the entire variable.  What do you think?